commit b63d4199f88e2b66896a053d6e1ac0b6442551bc (refs/changes/91/1591/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-02-28T14:33:24-08:00 (2 years, 7 months ago)
    
    DOCKER-1007 sdc-docker cli-filters.test.js tests failing with uknown '--last' argument

diff --git a/test/integration/cli-filters.test.js b/test/integration/cli-filters.test.js
index 27c0377..2caac65 100644
--- a/test/integration/cli-filters.test.js
+++ b/test/integration/cli-filters.test.js
@@ -132,12 +132,12 @@ test('container filters', function (tt) {
 
     // Filtering using limit - should match running/stopped containers.
     checkContainerFiltering(tt, '--filter name=' + CONTAINER_PREFIX
-        + ' --last 10',
+        + ' -n=10',
         [containerName1, containerName2]);
 
     // Filtering using negative limit - should only match running containers.
     checkContainerFiltering(tt, '--filter name=' + CONTAINER_PREFIX
-        + ' --last -10',
+        + ' -n=-10',
         [containerName2]);
 });
 
